packages/trader/src/Modules/SmartChart/Components/Markers/accumulators-profit-loss-tooltip.tsx

```diff
@@ -6,22 +6,18 @@ import { localize } from '@deriv/translations';
 import { FastMarker } from 'Modules/SmartChart';
 import { FastMarkerBeta } from 'Modules/SmartChartBeta';
 import AccumulatorsProfitLossText from './accumulators-profit-loss-text';
-import { ProposalOpenContract } from '@deriv/api-types';
-import { isMobile } from '@deriv/shared';
+import { getDecimalPlaces, isMobile } from '@deriv/shared';
```

Comment:
deprecated isMobile is replaced with is_mobile in this component in another PR: https://github.com/binary-com/deriv-app/pull/10848